modules/045-snapshot-controller/templates/snapshot-controller/deployment.yaml

```diff
@@ -0,0 +1,72 @@
+{{- if (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+---
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: d8-snapshot-controller
+  namespace: kube-system
+  {{- include "helm_lib_module_labels" (list . (dict "app" "d8-snapshot-controller" "workload-resource-policy.deckhouse.io" "master")) | nindent 2 }}
```

Comment:
Perhaps, a system node scheduling and workload-policy will do better?
You haven't specified any node scheduling constraints later on? Why do you specify `workload-resource-policy: master`?